content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -116,20 +115,21 @@ The reviews application (Java) does something similar:
 
 {{< text jzvz >}}
 @GET
-@Path("/reviews")
-public Response bookReviews(@CookieParam("user") Cookie user,
+@Path("/reviews/{productId}")
+public Response bookReviewsById(@PathParam("productId") int productId,
+                            @HeaderParam("end-user") String user,
                             @HeaderParam("x-request-id") String xreq,
                             @HeaderParam("x-b3-traceid") String xtraceid,
                             @HeaderParam("x-b3-spanid") String xspanid,
                             @HeaderParam("x-b3-parentspanid") String xparentspanid,
                             @HeaderParam("x-b3-sampled") String xsampled,
                             @HeaderParam("x-b3-flags") String xflags,
                             @HeaderParam("x-ot-span-context") String xotspan) {
-  String r1 = "";
-  String r2 = "";
+  int starsReviewer1 = -1;
+  int starsReviewer2 = -1;
 
-  if(ratings_enabled){
-    JsonObject ratings = getRatings(user, xreq, xtraceid, xspanid, xparentspanid, xsampled, xflags, xotspan);
+  if (ratings_enabled) {
+    JsonObject ratingsResponse = getRatings(Integer.toString(productId), user, xreq, xtraceid, xspanid, xparentspanid, xsampled, xflags, xotspan);
```

Comment:
These changes accurately reflect the code that is in the reviews service in the `release-1.0` branch.